src/main/frontend/common/i18n/i18n-provider.tsx

```diff
@@ -45,3 +45,7 @@ export const I18NProvider: FunctionComponent<I18NProviderProps> = ({
     <I18NContext.Provider value={messages}>{children}</I18NContext.Provider>
   );
 };
+
+export function useMessages() {
```

Comment:
does this need using elsewhere in a follow up?

Reply:
I was thinking of doing so